build.gradle.kts

```diff
@@ -74,6 +74,7 @@ kotlin {
         }
         val androidInstrumentedTest by getting {
             dependencies {
+                implementation(kotlin("test-junit"))
```

Comment:
Should this be a `testImplementation`? Or is this an artifact of how we are bundling kotlin + android libraries

Reply:
No dependencies declared in test source sets are included in the main artifact. `testImplementation` isn't available to use in that dependencies block.